Stride.BepuPhysics/Stride.BepuPhysics.csproj

```diff
@@ -29,6 +29,7 @@
   <ItemGroup>
     <PackageReference Include="BepuPhysics" Version="2.5.0-beta.15" />
     <PackageReference Include="Stride.Engine" Version="4.2.0.2043" />
+    <PackageReference Include="Stride.Physics" Version="4.2.0.2043" /><!--For precomputed hulls-->
```

Comment:
do the precomputed hulls rely on Bullet?

Reply:
Yes, and no. It is not part of the bullet DLL, it has its own, see https://github.com/stride3d/stride/tree/master/deps/VHACD and afair doesn't have any dependency on bullet.

Reply:
https://github.com/Nicogo1705/Stride.BepuPhysics/issues/11 Sweet! Ill do some testing and maybe we can close this easier than I thought.